tests/integrationv2/test_hello_retry_requests.py

```diff
@@ -138,3 +138,55 @@ def test_hrr_with_s2n_as_server(managed_process, cipher, provider, curve, protoc
     assert client_hello_count == 2
     assert server_hello_count == 2
     assert finished_count == 2
+
+# Default Keyshare for TLS v1.3 is x25519 
+TEST_CURVES = ALL_TEST_CURVES[1:] 
+@pytest.mark.uncollect_if(func=invalid_test_parameters)
+@pytest.mark.parametrize("cipher", TLS13_CIPHERS, ids=get_parameter_name)
+@pytest.mark.parametrize("provider", [OpenSSL])
```

Comment:
Can we also test with S2N, or do we not have the command line abilities to run this particular test?

Reply:
This test is intended to test s2nc and openssl s_server.